department-of-veterans-affairs/va.gov-cms
# VACMS-3703: Fix broken tab click behavior.

## Description
See #3703 

## Testing done
Visual

## Screenshots
N/A

## QA steps
As an administrator, go to `node/add/campaign_landing_page`
Click on any of the vertical tabs, and confirm that scrolling isn't frozen. Page shouldn't bounce or have any weird scroll behaviors.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
